deployment/src/main/java/org/jpristas/thesis/quarkus/multiconfig/deployment/propdoc/impl/writer/ProdProperty.java

```diff
@@ -0,0 +1,73 @@
+package org.jpristas.thesis.quarkus.multiconfig.deployment.propdoc.impl.writer;
+
+//import org.apache.commons.lang.StringUtils;
+
+public class ProdProperty {

```

Comment:
isnt this name a bit misleading -> "Prod"? I would add a comment why is it called like that. From the name of the class it seems that it is only relevant for prod environment